engine/src/main/java/org/destinationsol/game/drawables/RectSprite.java

```diff
@@ -70,8 +70,13 @@ public RectSprite(TextureAtlas.AtlasRegion tex, float texSz, float origPercX, fl
         isAdditive = additive;
     }
 
+    /**
+     * Sets the larger of the dimensions of the texture to the passed-in size, maintaining proportions and
+     * recalculating other fields as needed to fit.
+     * @param textureSize the new size for the sprite
+     */
     public void setTextureSize(float textureSize) {
-        textureSize /= level.depth;
+        textureSize /= level.depth; /** Scales the textureSize for background objects */
```

Comment:
This on the other hand would *not* be javadoc, since it is in the middle of a method. Good place for a plain old `//` comment at the end of the line, or on the preceding line.